dcicutils/beanstalk_utils.py

```diff
@@ -393,9 +393,9 @@ def _get_beanstalk_configuration_settings(env):
             }
     """
     try:
-        client = boto3.client('elasticbeanstalk')
+        client = boto3.client('elasticbeanstalk', region_name='us-east-1')
```

Comment:
I've noticed this is sometimes not passed.

1. Should we have a wrapper that always adds this?
2. Should we have a variable that holds the region name? It _might_ actually want to be a configuration param.

A comment noting possible future work would probably be fine here.